Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -1733,182 +1421,15 @@ private boolean updateBlocked() {
-         */
-        if (md.getRole().equals("Author")) {
-            Transliteration trans = new Transliteration();
-            try {
-                MetadataTypeInterface metadataTypeDIN = this.myPrefs.getMetadataTypeByName("AuthorTransliteratedDIN");
-                MetadataTypeInterface metadataTypeISO = this.myPrefs.getMetadataTypeByName("AuthorTransliteratedISO");
-                PersonInterface mdDin = UghImplementation.INSTANCE.createPerson(metadataTypeDIN);
-                PersonInterface mdIso = UghImplementation.INSTANCE.createPerson(metadataTypeISO);
-
-                mdDin.setFirstName(trans.transliterateDIN(md.getFirstName()));
-                mdDin.setLastName(trans.transliterateDIN(md.getLastName()));
-                mdIso.setFirstName(trans.transliterateISO(md.getFirstName()));
-                mdIso.setLastName(trans.transliterateISO(md.getLastName()));
-                mdDin.setRole("AuthorTransliteratedDIN");
-                mdIso.setRole("AuthorTransliteratedISO");
-
-                this.docStruct.addPerson(mdDin);
-                this.docStruct.addPerson(mdIso);
-            } catch (MetadataTypeNotAllowedException e) {
-                logger.error("Fehler beim Hinzufügen der Transliterationen (MetadataTypeNotAllowedException): "
-                        + e.getMessage());
-            }
-        }
-        saveMetadataAsBean(this.docStruct);
-
-        /* zum Schluss die Sperrung aktualisieren */
-        if (!updateBlocked()) {
-            return BLOCK_EXPIRED;
-        }
-        return "";
-    }
-
     /*
      * aus einer Liste von PPNs Strukturelemente aus dem Opac ## holen und dem
      * aktuellen Strukturelement unterordnen
      */
 
-    /**
-     * mehrere PPNs aus dem Opac abfragen und dem aktuellen Strukturelement
-     * unterordnen.
-     */
-    public String addAdditionalOpacPpns() {
-        StringTokenizer tokenizer = new StringTokenizer(this.additionalOpacPpns, "\r\n");
-        while (tokenizer.hasMoreTokens()) {
-            String tok = tokenizer.nextToken();
-            try {
-                FileformatInterface addRdf = CataloguePlugin.getFirstHit(opacCatalog,
-                    QueryBuilder.restrictToField(opacSearchField, tok), myPrefs);
-                if (addRdf != null) {
-                    this.docStruct.addChild(addRdf.getDigitalDocument().getLogicalDocStruct());
-                    readMetadataAsFirstTree();
-                } else {
-                    Helper.setMessage("Opac abgefragt: ", "kein Ergebnis");
-                }
-            } catch (TypeNotAllowedAsChildException | PreferencesException | RuntimeException e) {
-                logger.error(e.getMessage(), e);
-            }
-        }
-        return "Metadaten3links";
-    }
-
-    /**
-     * eine PPN aus dem Opac abfragen und dessen Metadaten dem aktuellen
-     * Strukturelement zuweisen.
-     */
-    public String addMetadaFromOpacPpn() {
```

Comment:
Will this functionality (Adding meta data from OPAC) added back in future?

Reply:
Yes